content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -206,6 +206,7 @@ kubectl create -f deploy/kubernetes/snapshot-controller
 | logFormat | CSI driver log format. Allowed values: "TEXT" or "JSON". | Yes | "TEXT" |
 | kubeletConfigDir | kubelet config directory path. Ensure that the config.yaml file is present at this path. | Yes | /var/lib/kubelet |
 | defaultFsType | Used to set the default FS type which will be used for mount volumes if FsType is not specified in the storage class. Allowed values: ext4, xfs. | Yes | ext4 |
+| fsGroupPolicy | Defines which FS Group policy mode to be used. Supported modes are`None, File, and ReadWriteOnceWithFSType.` | No | "ReadWriteOnceWithFSType" |
```

Comment:
In the values file the default is File.
https://github.com/dell/csi-powerflex/blob/main/helm/csi-vxflexos/values.yaml#L80